frameworks/kafka/docs/api-reference.md

```diff
@@ -400,7 +400,16 @@ Note the warning in the output from the commands above. You can change the indic
     {
         "message": ""
     }
-    
+
+# Service Status
```

Comment:
@sascala There are a bunch of endpoints that will be common across frameworks, within dcos-commons. The suppress endpoint is one example. We may want to consolidate the docs for those endpoints, so we don't have duplication. We may also want to point this fact out to users, so that they know they can use the same endpoint for different SDK frameworks. If it's easier, you could do this consolidation in a separate PR.